lecture_4/lecture_4_solution/test/components/Letter.spec.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import Letter from '../../js/components/Letter';
+
+describe('Letter', () => {
+
+  var Wrapper = React.createClass({
```

Comment:
Move wrapper to a reusable helper. Check lecture 5 examples where I have done so if necessary